examples/node_test/node_test.cpp

```diff
@@ -0,0 +1,162 @@
+//
+// Simple example to test out multiple nodes
+//
+// Author: Kalyan Sriram <coder.kalyan@gmail.com>
+
+#include <chrono>
+#include <cstdint>
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/action/action.h>
+#include <mavsdk/plugins/telemetry/telemetry.h>
+#include <iostream>
+#include <thread>
+
+using namespace mavsdk;
+using namespace std::this_thread;
+using namespace std::chrono;
+
+#define ERROR_CONSOLE_TEXT "\033[31m" // Turn text on console red
+#define TELEMETRY_CONSOLE_TEXT "\033[34m" // Turn text on console blue
+#define NORMAL_CONSOLE_TEXT "\033[0m" // Restore normal console colour
+
+void usage(std::string bin_name)
+{
+    std::cout << NORMAL_CONSOLE_TEXT << "Usage : " << bin_name << " <connection_url>" << std::endl
+              << "Connection URL format should be :" << std::endl
+              << " For TCP : tcp://[server_host][:server_port]" << std::endl
+              << " For UDP : udp://[bind_host][:bind_port]" << std::endl
+              << " For Serial : serial:///path/to/serial/dev[:baudrate]" << std::endl
+              << "For example, to connect to the simulator use URL: udp://:14540" << std::endl;
+}
+
+void component_discovered(ComponentType component_type)
+{
+    std::cout << NORMAL_CONSOLE_TEXT << "Discovered a component with type "
+              << unsigned(component_type) << std::endl;
+}
+
+int main(int argc, char** argv)
+{
+    Mavsdk dc;
```

Comment:
We should start calling this `mavsdk_server` instead of `dc` :sweat_smile:. `dc` is legacy from when the project was called "DroneCore" :laughing:

Reply:
Oh, I was curious where that came from! I'll change it :)